File: bachngocsach/src/genre.lua
local doc = http:get("https://bachngocsach.com/reader/theloai"):html()

local genre = {}
if doc ~= nil then
    local el = doc:select("div.view-content .theloai-row a")
    for i = 0, el:size() - 1 do
        local e = el:get(i)
        local link = {}
        link["title"] = e:text()
        link["input"] = "https://bachngocsach.com" .. e:attr("href")
        link["script"] = "gen.lua"
        table.insert(genre, link)
    end
    return response:success(genre)
end

return nil

File: bachngocsach/src/genre.js
load('config.js');
function execute() {
    let response = fetch(BASE_URL + "/reader/theloai");
    if (response.ok) {
        let doc = response.html();
        let genre = [];
        doc.select("div.view-content .theloai-row a").forEach(e => genre.push({
            title: e.text(),
            input: BASE_URL + e.attr("href"),
            script: "gen.js"
        }));
        return Response.success(genre);
    }

    return null;
}